core/src/main/scala/org/broadinstitute/dsde/rawls/jobexec/SubmissionMonitorActor.scala

```diff
@@ -748,7 +765,7 @@ trait SubmissionMonitor extends FutureSupport with LazyLogging with RawlsInstrum
           val cnt = attributeCount(update.upserts.values)
           logger.debug(
             s"Updating $cnt attribute values for entity ${update.entityRef.entityName} of type ${update.entityRef.entityType} in ${submissionId.toString}/${workflowRecord.externalId
-                .getOrElse("MISSING_WORKFLOW")}. ${safePrint(workspace.attributes)}"
+                .getOrElse("MISSING_WORKFLOW")}. ${safePrint(update.upserts)}"
```

Comment:
This is not strictly part of this PR, but as I was poring over logs, I noticed this problem. This log statement correctly outputted the count of attribute updates, but then output the contents of _workspace_ updates.